docs/source/method_cards/gyro_dropout.md

```diff
@@ -0,0 +1 @@
+../../../composer/algorithms/gyro_dropout/README.md
```

Comment:
This file should actually be a symbolic link to the README. Inside the folder `/docs/source/method_cards/`, @jelite  can you please run `ln -s ../../../composer/algorithms/gyro_dropout/README.md gyro_dropout.md`